news/2017-08-30-qat-gazette-august-2017.md

```diff
@@ -0,0 +1,41 @@
+# QAT Gazette - August 2017
+
+Hello!
+
+To those of you who have been following this blog at its’ [previous home](http://osuqat.tumblr.com/), welcome back! We’ve shuffled through a bit of a lean period on account of the summer holidays (or in my case, even more work than usual), but I am happy to bring you **the return of The QAT Gazette!** Feel free to skip down a little if you’re already familiar with our humble blog and want to get straight into the news, but if this is your first time and you’re wondering what all the fuss is about, read on!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+**A little over a year ago**, the Quality Assurance Team formed a blog to give people an idea of the work that goes on behind the scenes, the internal topics that come up for discussion, and our visions for shaping the future of the osu! mapping and modding community. There were highlights of upcoming projects and events, reviews of past occurrences such as Beatmap Nominator tests and the Best of 2016 awards, and sometimes opinion pieces simply giving our thoughts on various aspects of the community, the mapping meta, and so on. With the growth of the mapping and modding community over the past year, as well as the significant changes to the role of the QAT, more and more people rightfully ask, **“What the heck do you guys actually do now!?”** Hence the birth of regular updates, and now, a move into the mainstream for all to see!
```

Comment:
best to split paragraphs up so they're not such dense blocks.

new paragraph at 'there were highlights'

Reply:
Split it up, I'm used to scientific papers where paragraphs take up a whole page sometimes